molecule/source-upgrade/molecule.yml

```diff
@@ -39,6 +39,17 @@ provisioner:
     name: ansible-lint
 scenario:
   name: source-upgrade
+  test_sequence:
+    - lint
+    - dependency
+    - syntax
+    - create
+    - prepare
+    - converge
+    # - idempotence  # pip installing from source always replaces the installed version
+    - side_effect
+    - verify
+    - destroy
```

Comment:
It looks like we've re-added the destroy command to the final scenario per job, which complicates debugging.